src/background.ts

```diff
@@ -14,9 +14,9 @@ const players: Player[] = [
   },
   {
     controlQueries: {
-      next: '.control.control-next',
-      play: '.control.control-play',
-      prev: '.control.control-prev',
+      next: ".control.control-next, .player-controls .svg-icon-group-item:nth-of-type(5) button",
```

Comment:
The CI is failing on those lines https://travis-ci.org/GabrielDuarteM/webplayer-hotkeys/jobs/432784650#L3981, can you check it, please?

Probably just a `npm run lint -- --fix` will fix it...

Thanks!